src/videoCache.js

```diff
@@ -72,7 +72,7 @@ function wrapURI(uri, impTrackerURLs) {
  * @return {Object|null} - The payload to be sent to the prebid-server endpoints, or null if the bid can't be converted cleanly.
  */
 function toStorageRequest(bid, {index = auctionManager.index} = {}) {
-  const vastValue = bid.vastXml ? bid.vastXml : wrapURI(bid.vastUrl, bid.vastImpUrl);
+  const vastValue = getVastValue(bid);
```

Comment:
I think we need a better name for `vastValue`. VastValue is a very vague term. WHat does it even mean ? That vagueness is now spreading in the codebase. 

I think vastValue really means vast XML. so:

```suggestion
  const vastXml = getVastXml(bid);
```

Reply:
sure, agreed